far/FarRus.hlf.m4

```diff
@@ -5803,99 +5805,105 @@ $ #far:config System.Executor.UseAppPath#
 #Shift+Enter#, то для исполнения команды Far Manager запускает
 Проводник Windows, который всегда использует раздел #App Path# реестра.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Изменить этот параметр можно только через ~far:config~@FarConfig@.
 
 
 @System.Executor.ExcludeCmds
 $ #far:config System.Executor.ExcludeCmds#
- Параметр позволяет задавать набор команд которые будут сразу передаваться
-в %comspec% для выполнения, поиск в PATH и т.п. не будет произведён.
+ Этот строковый параметр определяет команды, которые будут немедленно
+переданы на исполнение командному процессору операционной системы
+(заданному переменной окружения #ComSpec#), не производя поиск
+в текущем директории, директориях, перечисленных в переменной окружения
```

Comment:
I suppose it should be either "текуще*й*" or "каталоге".

Reply:
Thanks. Replaced with `каталог` everywhere.

Somehow, to me `Директория` is the French government of 1795 - 1799, while `директорий` is a computer file directory. :) Will remember to avoid the word in the future.